### ISSUE TITLE
Convert URLs in body into HTML links

### ISSUE THREAD

**Issue by kristjanjansen** [tripikad/trip2#70]
Convert all the links to HTML links using ```getBodyAttribute()``` accessor. Remove Markdown support for now.

Here is the sample code:

```php

<?php

// Modified version of http://stackoverflow.com/a/5289151 and http://stackoverflow.com/a/12590772

$str = ' http://www.stackoverflow.com <a href="http://www.somesite.com">somesite</a> www.othersite.org ';

$pattern = "/(?i)\b((?:https?:\/\/|www\d{0,3}[.]|[a-z0-9.\-]+[.][a-z]{2,4}\/)(?:[^\s()<>]+|\(([^\s()<>]+|(\([^\s()<>]+\)))*\))+(?:\(([^\s()<>]+|(\([^\s()<>]+\)))*\)|[^\s`!()\[\]{};:'\".,<>?«»“”‘’]))(?![^<>]*>)/i";

$str = preg_replace($pattern, '<a href="$1">$1</a>', $str);

echo $str;

```

Extract this to a helper function ```str_links()``` or smth and put it to helpers file:

```
/app/helpers.php
```

and add following to ```composer.json```:

```json

"autoload": {
	"files": [
		"app/helpers.php"
	],
	...
}
```

See also https://laracasts.com/lessons/managing-mass-user-settings-part-2

at 14:35

**Comment by kristjanjansen** [comment]
It might make sense to do #207 first

**Comment by kristjanjansen** [comment]
Closed in #791
